### PR TITLE
chore: easy to understand dbt version checks

### PR DESCRIPTION
**Pull request by chamini2**

### CODE REVIEW

**Review by mederka — APPROVED** [review]
This is nice 😄

File: src/faldbt/lib.py [hunk 1]
```diff
@@ -49,9 +49,10 @@ def initialize_dbt_flags(profiles_dir: str):
     """
     args = FlagsArgs(profiles_dir, None)
     user_config = parse.get_dbt_user_config(profiles_dir)
-    try:
+
+    if DBT_VCURRENT.compare(DBT_V1) >= 0:
```
Comment (isidentical) [inline src/faldbt/lib.py:53]:
```suggestion
    if not IS_DBT_V0:
```

**Review by isidentical — APPROVED** [review]
Looks good (if it makes sense, maybe use the existing constant for this)

**Review by mederka — APPROVED** [review]
Nice 😂

**Pull request merged**